__tests__/endpoint.test.js

```diff
@@ -78,7 +77,58 @@ describe('/api/topics', ()=>{
                   .get('/not-a-api')
                   .expect(404)
                   .then((response) => {
-                expect(response.body.msg).toBe("Invalid api endpoint");
+                expect(response.body.msg).toBe("No match found");
+                  });
+              });
+        })
+
+        describe('/api/articles/:article_id', () => {
+            test('GET:200 sends a single article object to the client', () => {
+              return request(app)
+                .get('/api/articles/3')
+                .expect(200)
+                .then((response) => {
+             
+                  expect(typeof response.body.article).toBe('object');
+                });
+            });
+           
+          test("Should return a object with the properties author ,title, article_id, body. topic, created_at,  votes, article_img_url", () => {
+                  return request(app).get("/api/articles/4").expect(200).then(response => {
+
+                      const responseObj = response.body.article
+
+                      expect(Object.keys(responseObj).length).toBe(8);
+
+
+                expect(responseObj).toHaveProperty("author", expect.any(String));
+                  expect(responseObj).toHaveProperty("title", expect.any(String));
+                  expect(responseObj).toHaveProperty("article_id", expect.any(Number));
+                  expect(responseObj).toHaveProperty("body", expect.any(String));
+                  expect(responseObj).toHaveProperty("topic", expect.any(String));
+                  expect(responseObj).toHaveProperty("created_at", expect.any(String)); 
+                  expect(responseObj).toHaveProperty("votes", expect.any(Number));
+                  expect(responseObj).toHaveProperty("article_img_url", expect.any(String));
+                          
```

Comment:
These tests are expecting any values of the right type. How much confidence does this test give you that you've responded with the correct article? For example, Is any string ok for the title or should it be the one that matches the id provided?